src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -268,31 +280,48 @@ private function registerMetadataConfiguration(ContainerBuilder $container, arra
 
     private function getBundlesResourcesPaths(ContainerBuilder $container, array $config): array
     {
-        $bundlesResourcesPaths = [];
-
-        foreach ($container->getParameter('kernel.bundles_metadata') as $bundle) {
-            $paths = [];
+        $result = [];
+        $bundles = $container->getParameter('kernel.bundles_metadata');
+        foreach ($bundles as $bundle) {
             $dirname = $bundle['path'];
+            if ($config['resources']['autoload']) {
+                if (in_array($bundle['namespace'], $config['resources']['bundles']['exclude'])) {
+                    continue;
+                }
+            } else {
+                if (!in_array($bundle['namespace'], $config['resources']['bundles']['include'])) {
+                    continue;
+                }
+            }
+            $result = $this->setResourcesPath($dirname, $container, $config);
+        }
+        return $result;
+    }
+
+    private function setResourcesPath($dirname, ContainerBuilder $container, array $config): array
+    {
+        $paths = [];
+        $result = [];
+        if ($config['resources']['sources']['configs']) {
             foreach (['.yaml', '.yml', '.xml', ''] as $extension) {
                 $paths[] = "$dirname/Resources/config/api_resources$extension";
             }
-            if ($this->isConfigEnabled($container, $config['doctrine'])) {
-                $paths[] = "$dirname/Entity";
-            }
-            if ($this->isConfigEnabled($container, $config['doctrine_mongodb_odm'])) {
-                $paths[] = "$dirname/Document";
-            }
-
-            foreach ($paths as $path) {
-                if ($container->fileExists($path, false)) {
-                    $bundlesResourcesPaths[] = $path;
-                }
+        }
+        if ($config['resources']['sources']['doctrine']) {
+            $paths[] = "$dirname/Entity";
```

Comment:
You can change this in doctrine settings

Reply:
Are you sure you understand the essence of the changes?
https://github.com/api-platform/core/blob/5b2cfdcd8feea9b9ff96371df1a7eeccc72432f4/src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php#L309
This is the code from the master. Why should I change the settings of the doctrine if I need to influence the settings of the Platform?